x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/support/ApiKeyBoolQueryBuilder.java

```diff
@@ -108,136 +95,6 @@ public static ApiKeyBoolQueryBuilder build(
         return finalQuery;
     }
 
-    private static QueryBuilder doProcess(QueryBuilder qb, Consumer<String> fieldNameVisitor) {
```

Comment:
This is the method that I need to expose publicly (to be used to "translate" the queries used by the aggs builder in https://github.com/elastic/elasticsearch/pull/104895).